src/main/java/org/folio/rest/impl/InvoiceLineHelper.java

```diff
@@ -449,7 +455,7 @@ private Future<List<InvoiceLine>> applyProratedAdjustments(InvoiceLine invoiceLi
       return adjustmentsService.applyProratedAdjustments(allLines, invoice)
         .stream()
         .filter(line -> !line.equals(invoiceLine))
-        .collect(toList());
+        .toList();
```

Comment:
when making such changes please check that returned list is unmodifiable list, we need to make sure that the code not modify this collection

Reply:
Thanks to remind. reverted some of them just in case